NuGet.Config

```diff
@@ -1,11 +1,12 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-  <packageSources>
-    <clear />
-    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-  </packageSources>
-  <config>
-    <add key="globalPackagesFolder" value="./packages" />
-    <add key="repositoryPath" value="./packages" />
-  </config>
+	<packageSources>
+		<clear />
+		<add key="localpackages" value=".\refs\localpackages" />
```

Comment:
><add key="localpackages" value=".\refs\localpackages" /> [](http://example.com/codeflow?start=2&length=56)

this is a temporary change,
the reason is Json Schema reference Sarif SDK and Sarif SDK reference Json Schema, without update both at the same time, the build and test will not be able to pass.
this local nuget store contains local build of the Json Schema.
after this we can publish a new version, 
then update the Json Schema to use the new version,
then finally update the Sarif SDK to use the new vesion of the Json Schema nuget, and this local nuget store can be removed at the same changeset.

Reply:
the Json Schema PR will be about the same as this one but it will be after the Sarif SDK publish a new version, to use the latest Sarif SDK nuget. 
[https://github.com/microsoft/jschema/pull/139](https://github.com/microsoft/jschema/pull/139)

Reply:
just wondering about the local package thing.
why there is a reference loop between Sarif SDK and Json Schema? can we break the loop to avoid doing this temporary thing?

Reply:
Sarif SDK solution use Json Schema to validate if a json is a valid Sarif file,
in Json Schema the method to validate json use Sarif SDK to maintain a Sarif format result in memory and later to output.
the property we are changing "level" is a property used explicitly in both place.